test/browser/test/happy.spec.js

```diff
@@ -0,0 +1,132 @@
+import { test, expect } from "@playwright/test";
+
+test.describe("browser tests", function () {
+  let tableName;
+
+  test("has title", async function ({ page }) {
+    await page.goto("http://localhost:5173/");
+
+    // Expect a title "to contain" a substring.
+    await expect(page).toHaveTitle(/Tableland SDK Test App/);
+  });
+
+  test("can create", async function ({ page }) {
+    await page.goto("http://localhost:5173/");
+
+    const success = "table was created";
+    await page.type("input[name=statement]", "CREATE TABLE browser_table (k text, v text, num integer);");
+    await page.type("input[name=success]", success);
+    await page.click("input[name=submit-sql]");
+
+    await expect(page.getByTestId("status")).toHaveText(success);
```

Comment:
The Playwright `expect` test assertion works internally by constantly polling the page's html until either it matches the expectation, or the config timeout is reached.